src/interface/web/app/chat/page.tsx

```diff
@@ -29,22 +29,37 @@ interface ChatBodyDataProps {
     isLoggedIn: boolean;
 }
 
+var image64 = "";
+
 function ChatBodyData(props: ChatBodyDataProps) {
     const searchParams = useSearchParams();
     const conversationId = searchParams.get("conversationId");
     const [message, setMessage] = useState("");
+    const [image, setImage] = useState<string | null>(null);
     const [processingMessage, setProcessingMessage] = useState(false);
     const [agentMetadata, setAgentMetadata] = useState<AgentData | null>(null);
 
     const setQueryToProcess = props.setQueryToProcess;
     const onConversationIdChange = props.onConversationIdChange;
 
+    useEffect(() => {
+        if (image) {
+            image64 = encodeURIComponent(image);
```

Comment:
one small thing about the feedback surrounding the image64 as a state variable. The problem is that it is being used by too fairly independent methods so I think it might make more sense for it to be a page level variable rather than trying to make it a state var

Reply:
done!